File: Python/Ex_37.py
values = []
max = int(input('How many values do you want to type? (max of 20): '))

while max > 20:
    max= int(input('Error! The typed value is out of range.\n Please enter the amount again: '))

for c in range(0, max):
    v = float(input("Enter with a value: "))
    values.append(v)

answer = "Y"

while answer != "N":

    search = int(input('Enter with the value you want to search for: '))

    if search in values:
        position = values.index(search)
        print(f"The position is {position}")
    else: 
        print("Value not found!")
    while search not in values:
        answer = input('Do you want to make another search? (y/n): ').upper()
        if answer == "Y":
            search = int(input('Enter with the value you want to search for: '))
    answer = input('Do you want to make another search? (y/n): ').upper()

File: c/Ex_37.c
#include <stdio.h>
#include <stdlib.h>
#include <ctype.h>

int main(int argc, char const *argv[])
{
    int max, search, t;
    int values[20];


    printf("How many values do you want to type? (max of 20): \n");
    scanf("%i", &max);
 
    for(int i = 0; i < max; i++)
    {
        printf("\nEnter with a value: \n");
        scanf("%i", &values[i]);      
    }

    searching:

    printf("\nEnter with the value you want to search for: \n");
    scanf("%i", &search);

    for (int i = 0; i < max; i++)
    {
        if (values[i] == search)
        {
            printf("\n\n The position is %i\n\n", i);
        }
        else
        {
            printf("\n\nValoue not found");
        }
    } 

    printf("\n\nIf you want to search for another number type '1' otherwise type '2':\n");
    scanf("%s", & t);

    if (t==1)
    {
        goto searching;
    }
    else
    {
        printf("\nok");
    }
}